src/lib/core/CHIPConfig.h

```diff
@@ -1390,11 +1390,18 @@ extern const char CHIP_NON_PRODUCTION_MARKER[];
 #define CHIP_CONFIG_SCENES_CLUSTER_MAXIMUM_NAME_LENGTH 16
 #endif
 
+/**
+ * @brief Maximum transition time for scenes in seconds
+ */
+#ifndef CHIP_CONFIG_SCENES_MAX_TRANSITION_TIME_SECONDS
+#define CHIP_CONFIG_SCENES_MAX_TRANSITION_TIME_SECONDS 6000
```

Comment:
Why is this an application config value?  This is a spec-defined constant.  It should just be a constant in the code, not configurable so applications can get it wrong.